docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+
+| Plan         | High availability features                                      | Backup history | Uptime  |
+| ------------ | --------------------------------------------------------------- | -------------- | ------- |
+| **Hobbyist** | Single node with limited availability                           | 2 days         | 98.5 %  |
+| **Startup**  | Single node with limited availability                           | 2 days         | 99 %    |
+| **Business** | Two nodes (primary + standby: basic high availability)          | 14 days        | 99.9 %  |
+| **Premium**  | Three nodes (primary + 2 x standby: stronger high availability) | 30-day         | 99.99 % |
+
+## About primary and standby nodes
+
+Aiven's Business and Premium plans offer primary nodes and standby nodes. A standby
+service is useful for multiple reasons:
+
+-   Provides another physical copy of the data in case of hardware,
+    software, or network failures
+-   Typically reduces the data loss window in disaster scenarios
+-   Provides a quicker database time to restore with a controlled
+    failover in case of failures, as the standby is already installed,
+    running, and synchronised with the data
+-   Can be used for read-only queries to reduce the load on the primary
+    server
+
+## Failure handling
+
+### Minor failures
+
+Minor failures, such as service process crashes or temporary loss of
+network access, are handled automatically by Aiven in all plans without
+any major changes to the service deployment. The service automatically
+restores normal operation once the crashed process is restarted or when network access is
+restored.
+
+### Severe failures
+
+Severe failures, such as losing a node entirely in case of hardware
+or severe software problems, require involved recovery measures. The
+Aiven monitoring infrastructure automatically detects a failing node
+both when the node starts reporting issues in the self-diagnostics or
+when stops communicating. In such cases, the monitoring infrastructure
+automatically schedules a new replacement node to be created.
+
+:::note
+In the event of database failover, the **Service URI** of your service
+remains the same; only the IP address will change to point to the new
+primary node.
+:::
+
+## Highly available Business and Premium service plans
+
+When the failed node is a MySQL standby node, the primary node
+keeps running normally and provides a normal service level to the client
+applications. Once the new replacement standby node is ready and
+synchronised with the primary node, it starts replicating the primary
+node in real time as the situation reverts back to normal.
+
+When the failed node is a MySQL primary node, the combined
+information from the Aiven monitoring infrastructure and the standby
+node is used to make a failover decision. The standby node is then
+promoted as the new primary and immediately starts serving clients. A
```

Comment:
I would nuance "immediately" here.  If there is replication lag, this will not apply (or we might promote the node immediately, but then loose unapplied trx).